gists/helm-component-extraction/promtail/README.md

```diff
@@ -0,0 +1,40 @@
+# Promtail
+
+## Chart details
+
+| Chart                                                           | Version  | App version |
+| --------------------------------------------------------------- | -------- | ----------- |
+| [source](https://artifacthub.io/packages/helm/grafana/promtail) | `v6.6.1` | `v2.6.1`    |
+
+## Prerequisites
+
+### Log into AWS
+
+1. Export the `AWS_PROFILE` variable with the relevant profile name.
+    ```bash
+		export AWS_PROFILE=your-profile-name
+		```
+2. Log in to the AWS account with `aws sso login`
+
+### Log into okctl
+
+1. Run `okctl venv -c <path to relevant cluster.yaml>` to log into the okctl cluster.
+
+## Uninstall okctl provisioned Promtail
+
+1. Edit the relevant `cluster.yaml` file and set `integrations.promtail` to `false`.
```

Comment:
Note: Make sure that no integrations are still commented out if you were running on with everything default in your cluster file.